Dockerfile

```diff
@@ -0,0 +1 @@
+FROM golang:1.18.0
```

Comment:
What is this Dockerfile for?

Reply:
I don't think the Docker flow runs unless there is a Dockerfile, no?

Reply:
Having a compose test file, or a list of docker repos for publishing is the requirement. Flowzone doesn't look for Dockerfiles.

Reply:
Without, I get this: https://github.com/balena-io/sshproxy/actions/runs/4876120556/jobs/8699234734#step:18:207

Reply:
Ah yes, that looks like a bug in Flowzone. We assume if you have compose tests you must want to bake something first.

Opened a new issue: https://github.com/product-os/flowzone/issues/628

I guess you need the Dockerfile for now.

Reply:
Okey dokey, I'll put it back.